sklearn/ensemble/_bagging.py

```diff
@@ -523,6 +533,10 @@ class BaggingClassifier(ClassifierMixin, BaseBagging):
     n_features_ : int
         The number of features when :meth:`fit` is performed.
 
+        .. deprecated:: 1.0
+            Attribute n_features_ was deprecated in version 1.0. and will be
```

Comment:
I think sphinx is interpreting this as a link. We can force it to be code like so:

```suggestion
            Attribute `n_features_` was deprecated in version 1.0. and will be
```

Reply:
Thanks :pray: